Packs/HarfangLabEDR/pack_metadata.json

```
@@ -1,11 +1,11 @@
 {
     "name": "HarfangLab EDR",
     "description": "This connector allows to fetch security events from a HarfangLab EDR Manager and manage the incident response.",
-    "support": "community",
-    "currentVersion": "1.0.0",
-    "author": "Pierre-HarfangLab",
-    "url": "",
-    "email": "",
+    "support": "partner",
+    "currentVersion": "1.0.1",
```

```suggestion
    "currentVersion": "1.0.0",
```
metadata and README changes do not require version bump